satellite/admin/README.md

```diff
@@ -54,6 +54,10 @@ Updates usage limit for a project.
 
 Updates rate limit for a project.
 
+## DELETE /api/project/{projectid}
```

Comment:
The convention in this README seems to be `{project-id}` instead of {projectid}.

Reply:
Oh, i copy pasted from 3887, where i changed the convention to be more straight. Lemme change it back for now.